modules/programs/beets.nix

```diff
@@ -28,6 +28,19 @@ in
         '';
       };
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.beets;
+        defaultText = literalExample "pkgs.pass";
```

Comment:
I recently learned that this actually shouldn't be necessary: During manual building NixOS (and home-manager too) replace `pkgs.foo` with `"pkgs.foo"`. I'd test it, but I think this line can just be removed.

Reply:
Done

Reply:
Hm actually, after trying this out myself, this doesn't seem to do what I expected it to do. Instead of what I said above, it gets replaced with just `pkgs.beets.outPath`, which is some /nix/store path, but without depending on it. It shows up in the manpage as

![2019-12-18_00-58](https://user-images.githubusercontent.com/20525370/71044285-8e231780-2131-11ea-82ff-b594088cd96a.png)

So TIL, and correction: This should use `defaultText` after all (but this time with `beets` and not `pass` :))

Reply:
I used `literalExample` please let me know if you really meant `defaultText`.